brain/files.cmake

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2013-2015, EPFL/Blue Brain Project
```

Comment:
For brion it's separated and it's quite long, so that one should be kept separate. I did the same here for consistency.